geode-core/src/test/java/org/apache/geode/test/process/ProcessWrapper.java

```diff
@@ -391,7 +391,7 @@ private void start(final Properties properties, final File workingDirectory) {
 
     // -d64 is not a valid option for windows and results in failure
     final int bits = Integer.getInteger("sun.arch.data.model", 0).intValue();
-    if (bits == 64 && !(System.getProperty("os.name").toLowerCase().contains("windows"))) {
+    if (bits == 64 && (System.getProperty("os.name").toLowerCase().contains("solaris"))) {
```

Comment:
Does this mean that "-d64" is no longer a valid option for Mac or Linux?

Reply:
I think we can get rid of this option for all the platforms. This is from java9 documentation:

-d32
This option is deprecated and will be removed in a future release.

-d64
This option is deprecated and will be removed in a future release.

Oracle Solaris, Linux, and OS X: Runs the application in a 64-bit environment. If a 64-bit environment isn’t installed or isn’t supported, then an error is reported.

Reply:
Should we just rip this out entirely?  Who's running a 32bit OS anyway?